packages/react-dom/src/__tests__/ReactDOMRoot-test.js

```diff
@@ -20,52 +21,56 @@ describe('ReactDOMRoot', () => {
     container = document.createElement('div');
   });
 
-  it('renders children', () => {
-    const root = ReactDOM.createRoot(container);
-    root.render(<div>Hi</div>);
-    expect(container.textContent).toEqual('Hi');
-  });
+  if (ReactFeatureFlags.enableCreateRoot) {
```

Comment:
@clemmy @gaearon Can y'all explain how this works? Do we run the tests twice?

Reply:
I'm not sure what you are asking. My understand is that the tests should only run once, or not at all.